[DOCS] [8.0-8.2] Add release note for networking on macOS with Endpoint enabled (#1994)
elastic/security-docs: docs/release-notes/8.0.asciidoc

** If you don't want to preserve the comment, remove the comment entirely.
. Save the file and re-import it.

*Network connection issues might occur if {elastic-endpoint} is used with network traffic tools*

> If {elastic-endpoint} is used with other products that monitor or manage network traffic (such as antivirus programs, firewalls, or VPNs) users might experience network connection issues. This issue is only present on macOS; upgrading to macOS 12.4 resolves it.

Author: ```suggestion
If {elastic-endpoint} is used with other products that monitor or manage network traffic (such as antivirus programs, firewalls, or VPNs) users might experience network connection issues. This issue is only present on macOS. To resolve it, upgrade to macOS 12.4.
```

> If {elastic-endpoint} is used with other products that monitor or manage network traffic (such as antivirus programs, firewalls, or VPNs) users might experience network connection issues. This issue is only present on macOS. To resolve it, upgrade to macOS 12.4.

Reviewer: ```suggestion
On MacOS versions before 12.4, if {elastic-endpoint} is used with other products that monitor or manage network traffic (such as antivirus programs, firewalls, or VPNs) users might experience network connection issues. 
```
This way people who don't use old MacOS would be able to skim more easily. If you like the suggestion it applies below as well.

Author: I like it @benironside! I do want to mention that users should upgrade to macOS 12.4 though since that's the "workaround" to this issue.

Author: ```suggestion
On MacOS versions before 12.4, if {elastic-endpoint} is used with other products that monitor or manage network traffic (such as antivirus programs, firewalls, or VPNs) users might experience network connection issues. To resolve this issue, upgrade to macOS 12.4.
```

Reviewer 2: Combines edits from @benironside and @nastasha-solomon, plus a couple more ("macOS" starts lowercase, and I assume future macOS releases will also resolve the issue):
```suggestion
On macOS versions before 12.4, if {elastic-endpoint} is used with other products that monitor or manage network traffic (such as antivirus programs, firewalls, or VPNs), users might experience network connection issues. To resolve this issue, upgrade to macOS 12.4 or later.
```

Author: Ah, good point. I think it's fairly safe to assume the fix implemented in macOS 12.4 was carried forward. Thanks for pointing that out @joepeeples !
